crates/nu-path/src/helpers.rs

```diff
@@ -1,13 +1,41 @@
+use etcetera::BaseStrategy;
 #[cfg(windows)]
 use omnipath::WinPathExt;
+use once_cell::sync::Lazy;
 use std::path::PathBuf;
 
+static HOME_DIR: Lazy<Option<PathBuf>> = Lazy::new(|| etcetera::home_dir().ok());
+static CONFIG_DIR: Lazy<Option<PathBuf>> = Lazy::new(|| {
+    let default = if let Some(mut config_dir) = home_dir() {
+        config_dir.push(".config");
+        if config_dir.join("nushell").is_dir() {
+            return Some(config_dir);
+        }
+        Some(config_dir)
+    } else {
+        None
+    };
+
+    if let Ok(basedirs) = etcetera::base_strategy::choose_native_strategy() {
```

Comment:
This par seems reasonable but it would be nice to have some comments explicitly stating where these locations are so the next person doesn't have to spend so much time looking this up. This is what I found, hopefully it's right.
```
HOME_DIR
apple: HOME env var
linux: HOME env var
windows: USERPROFILE env var

CONFIG_DIR default
apple: HOME_DIR/.config/nushell if HOME_DIR is set, otherwise None
linux: HOME_DIR/.config/nushell if HOME_DIR is set, otherwise None
windows: USERPROFILE/AppData/Roaming/nushell <-- this needs to be changed as per my comment above.

base_strategy.native_strategy
apple: HOME_DIR/Library/Application Support/nushell
linux: HOME_DIR/.config/nushell
windows: USERPROFILE/AppData/Roaming/nushell 
```

Reply:
Although I've not tested this myself, the logic should work on Windows too.

Reply:
Thanks for the additional comments.